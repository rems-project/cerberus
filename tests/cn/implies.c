int identity(int x) 
{
    int y = x;
    /*@ assert((x == 0i32) implies (y == 0i32));@*/
    return y;
}