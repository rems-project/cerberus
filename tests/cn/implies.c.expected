return code: 0
[1/1]: identity -- pass
